spec/features/merchant_dashboard/items_index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe "as a merchant employee, when i visit my merchant dashboard", type: :feature do
+  it "i see a link to my items and when i click it i go to /merchant/items" do
+
+    meg = Merchant.create!(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+    tire = meg.items.create!(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+    pull_toy = meg.items.create!(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
+    dog_bone = meg.items.create!(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+    user = meg.users.create!(
+      name: "Steve",
+      address:"123 Main St.",
+      city: "Fort Collins",
+      state: "GA",
+      zip: "66666",
+      email: "chunky_lover@example.com",
+      password: "123password",
+      role: 1
+    )
+
+    visit "/login"
+    fill_in :Email, with: user.email
+    fill_in :Password, with: user.password
+    click_button "Login"
+
+    expect(current_path).to eq("/merchant/dashboard")
+
+    click_link("Merchant Items")
+
+    expect(current_path).to eq("/merchant/items")
+
+    expect(page).to have_content(tire.name)
+    expect(page).to have_content(pull_toy.name)
+    expect(page).to have_content(dog_bone.name)
```

Comment:
having already verified links, images, description, etc... i just wanted to verify the item showed up so just testing for name here.

Reply:
I think that testing for the item name is sufficient here!

Reply:
agreed with Steve